presto-cache/src/main/java/com/facebook/presto/cache/alluxio/AlluxioURIStatus.java

```diff
@@ -34,4 +34,14 @@ public HiveFileContext getHiveFileContext()
     {
         return hiveFileContext;
     }
+
+    public void recordBytesReadFromCache(long bytes)
```

Comment:
I think we should discuss this. This does not seem correct interface to me.

Reply:
I wanted to message in the Alluxio PR, I think we need to discuss with Alluxio how to populate this correctly.

Reply:
Will discuss this with Alluxio team in our weekly sync meeting.